appserver/admingui/cluster/src/main/resources/cluster/clusterGeneral.jsf

```diff
@@ -127,6 +127,17 @@
         />
     </sun:button>
 
+    <sun:button id="restartButton" text="Restart Cluster" disabled="#{disableStop}" primary="#{false}"
```

Comment:
Set the `disabled` boolean to the value of `disableStop` since we want the restart button enabled at the same times as the stop button.